.travis.yml

```diff
@@ -4,60 +4,81 @@ addons:
   homebrew:
     packages:
     update: false
+
 stages:
-  - "Miscellaneous"
+  - name: "No dependencies" # Doesn't depend on anything apart from upstream packages
+    if: type != cron
```

Comment:
How often does the cron run? It might make sense to run the build regularly anyway?